x/multi-staking/keeper/keeper_test.go

```diff
@@ -14,6 +14,7 @@ import (
 type KeeperTestSuite struct {
 	suite.Suite
 
+	app      *simapp.SimApp
```

Comment:
I see that we use msgServer a lot in testing. We should declear it in test suite